helm-flx.el

```diff
@@ -148,7 +148,7 @@ Return candidates prefixed with basename of `helm-input' first."
     (if (string= helm-pattern "")
         candidates
       (if (string-match-p " " helm-pattern)
-          (helm-fuzzy-matching-default-sort-fn candidates source use-real)
+          (helm-fuzzy-matching-default-sort-fn candidates source)
         (helm-flx-sort candidates
                        helm-pattern
                        (if use-real
```

Comment:
> If you mean line 154 `(if use-real`

Yes, that what I meant.

> I didn't touch that because it's an argument to `helm-flx-sort`; if we remove that, it might break `helm-flx-sort`, and I'm not familiar enough with the code to feel comfortable messing with that.

Fair enough. I'm not that familiar with the code either.

>  This should fix the breakage that's affecting every Spacemacs user

More accurately, every user of `helm-flx` that updated their Helm packages recently (`helm-core` specifically, since ~2 days ago I think)